scripts/uitest.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+set -e
+
+# Define paths to uitest directories and command.
+script_dir=`dirname $0`
+root_dir=`realpath $script_dir/../`
+uitest_dir=$root_dir/uitest
```

Comment:
I'll likely have a version of uitest already checked out which may or may not have modifications I'd like to use. Should this be configurable? Or maybe instead be `$root_dir/../`

Also I'd prefer constants to be uppercase :) but up to you.